ledger/test-common/src/main/daml/simple_package_optional/Simple.daml

```diff
@@ -0,0 +1,30 @@
+module Simple where
+data SimpleVariant = SV Party
+template SimpleTemplate
+   with
+     owner : Party
+     observer : Party
+     contractData : Optional Party
```

Comment:
Spotting the difference between those files is quite hard and keeping them in sync even more so. Can we instead have one template and throw `sed` or something at it to replace this one type?

Reply:
After simplifying them with `Archive` it should be easier. I don't think deduplicating them would be trivial, given the current shape of the Bazel files. Let me know if you think it's still worth spending more time on it. I would leave it as is, now.